src/Editor/EditorUtils.js

```diff
@@ -27,12 +27,30 @@ export const displayTextWithMentions = (inputText, formatMentionNode) => {
             });
         }else{
             formattedText.push(retLine);
-        }            
+        }
         formattedText.push("\n");
-    });    
+    });
     return formattedText;
 }
 
+export const formattedTextToPlain = (inputText) => {
+  let output = '';
+  const mentions = EU.findMentions(inputText)
+  if(mentions.length  === 0) {
+    return inputText
+  }
+  let lastIdx = 0;
```

Comment:
This is not really the last index, but the next one. or current one.

it would be correct to call it `lastIdx` if it were one less, that is the last index which was copied to the result.

Reply:
You're right, this is used to refer to the current index being considered as the start of the following segment. Will rename